lib/resources/nginx_module.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: utf-8
+# author: Rony Xavier, rx294@gmail.com
+# author: Aaron Lippold, lippold@gmail.com
+# license: All rights reserved
```

Comment:
Please remove this line. InSpec is released under Apache 2.0. We have a pending PR to remove this line from all our files (acknowledging that you probably copy/pasted this from another existing file 🙂 ).